test/argent-account-test.ts

```diff
@@ -107,14 +107,18 @@ describe("Argent account", function () {
         const { res: initialBalance } = await mainContract.call("get_balance");
 
         const newGuardianPrivateKey = "0x123";
-        await account.setGuardian(newGuardianPrivateKey);
+        await account.setGuardian(newGuardianPrivateKey, {
+            maxFee: 1e18
```

Comment:
Why is this necessary all of a sudden?

Reply:
setGuardian doesn't have an implicit maxFee calculation and error `message":"max_fee == 0 is not supported` will be thrown if it's not passed with starknet v0.11.0